### DISCUSSION TITLE
How should I modify the loading model code to fine tune my pretrained weights?

### DISCUSSION THREAD

**Discussion by ykemiche** [facebookresearch/detectron2#2956] [Q&A]
Hello ! 

After fine tuning a model from model zoo,I got "model_final.pth",Now I want to fine tune this  model("model_final.pth") in another task :

-How can I modify the piece of code bellow to load my model(I just trained) and its configurations?

```
cfg = get_cfg()
cfg.merge_from_file(model_zoo.get_config_file("COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml"))
cfg.DATASETS.TRAIN = ("train",)
cfg.DATASETS.TEST = ()
cfg.DATALOADER.NUM_WORKERS = 2
cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url("COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml")
```

-What kind of information does exactly the ".yaml" file contain?

Thanks in advance

**Comment by shauryagoel** [comment]
Did you solve this problem?

**Comment by zhuimeng2080** [comment]
given the trained model on custom datasets "model_final.pth",  how to fine tune it on one 

specific class of small new datasets
